tests/postgres/ttables.nim

```diff
@@ -56,7 +56,7 @@ suite "Table creation":
 
     check dbConn.getAllRows(qry, "FurnitureTable") == @[
       @[?"id", ?"bigint"],
-      @[?"legcount", ?dftDbInt]
+      @[?"legCount", ?dftDbInt]
```

Comment:
An example for sudden case-sensitivity, though that is when comparing column names and not during actual queries, not sure how much that matters.